onnx_tf/backends/backend_v1.py

```diff
@@ -789,8 +789,9 @@ def handle_slice(cls, node, input_dict):
 
     for i in range(slice_len):
       ends[i] = full_sizes[axes[i]] + ends[i] if ends[i] < 0 else ends[i]
-      ends[i] = np.min([full_sizes[axes[i]], ends[i]])
-      starts[i] = np.min([full_sizes[axes[i]], starts[i]])
+      if full_sizes[axes[i]] is not None:
```

Comment:
Do we need to emit an error if None is encountered?

Reply:
Even if None, we can also do slice with out of bounds risk.

Reply:
@tjingrant Sure.

Reply:
Any updates?

Reply:
I guess this seems to be the right thing to do for now to solve your problem. But let's discuss under your PR to see if there's anything high-level that needs to be done as well.